[xpu][feature][static_launcher] 4/4: Enable static triton kernel launcher for XPU backend.

etaf opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #171450
* #171432
* __->__ #169938
* #168952
* #169121

This PR enables the static launcher for XPU backend and reuse the corresponding UT.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @kadeng @muchulee8 @amjames @chauhang @aakhundov @coconutruben @jataylo

etaf committed:
[xpu][feature][static_launcher] 4/4: Enable static triton kernel launcher for XPU backend.

[ghstack-poisoned]

etaf committed:
Update on "[xpu][feature][static_launcher] 4/4: Enable static triton kernel launcher for XPU backend."


This PR enables the static launcher for XPU backend and reuse the corresponding UT.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo

[ghstack-poisoned]

etaf committed:
Update on "[xpu][feature][static_launcher] 4/4: Enable static triton kernel launcher for XPU backend."


This PR enables the static launcher for XPU backend and reuse the corresponding UT.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo

[ghstack-poisoned]

etaf committed:
Update on "[xpu][feature][static_launcher] 4/4: Enable static triton kernel launcher for XPU backend."


This PR enables the static launcher for XPU backend and reuse the corresponding UT.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo

[ghstack-poisoned]

etaf committed:
Update on "[xpu][feature][static_launcher] 4/4: Enable static triton kernel launcher for XPU backend."


This PR enables the static launcher for XPU backend and reuse the corresponding UT.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo

[ghstack-poisoned]

etaf committed:
Update on "[xpu][feature][static_launcher] 4/4: Enable static triton kernel launcher for XPU backend."


This PR enables the static launcher for XPU backend and reuse the corresponding UT.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo

[ghstack-poisoned]

etaf committed:
Update on "[xpu][feature][static_launcher] 4/4: Enable static triton kernel launcher for XPU backend."


This PR enables the static launcher for XPU backend and reuse the corresponding UT.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo

[ghstack-poisoned]

etaf committed:
Update on "[xpu][feature][static_launcher] 4/4: Enable static triton kernel launcher for XPU backend."


This PR enables the static launcher for XPU backend and reuse the corresponding UT.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo

[ghstack-poisoned]

etaf committed:
Update on "[xpu][feature][static_launcher] 4/4: Enable static triton kernel launcher for XPU backend."


This PR enables the static launcher for XPU backend and reuse the corresponding UT.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo

[ghstack-poisoned]

etaf commented:
@pytorchbot merge

yangw-dev commented:
@pytorchbot revert -m "the bottom stack break internal system, and casued SEV, please reachout internal staff for more details" -c ghfirst

etaf committed:
Update on "[xpu][feature][static_launcher] 4/4: Enable static triton kernel launcher for XPU backend."


This PR enables the static launcher for XPU backend and reuse the corresponding UT.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo

[ghstack-poisoned]

etaf committed:
Update on "[xpu][feature][static_launcher] 4/4: Enable static triton kernel launcher for XPU backend."


This PR enables the static launcher for XPU backend and reuse the corresponding UT.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo

[ghstack-poisoned]
